lib/libspl/zone.c

```diff
@@ -27,6 +27,8 @@
 #include <string.h>
 #include <errno.h>
 
+int aok;
```

Comment:
Let's explicitly initialize this to zero for readability here and in `edonr_test.c`.  `int aok = 0;`

Reply:
Done.